### DISCUSSION TITLE
Tabs on List View are loading too slow..

### DISCUSSION THREAD

**Discussion by sluxzinc92** [filamentphp/filament#9635] [Help]
Hello, please help me. On my live hosting, selecting another tab is taking too long unlike opening directly the link for that filtered tab. Is there a way where it won't use the livewire? just the link? cause as I can see it, livewire is the one taking too long to process.. 
![image](https://github.com/filamentphp/filament/assets/79960348/38d6fe5c-83dd-4991-916b-227b63bb0e73)
![image](https://github.com/filamentphp/filament/assets/79960348/f71a79dc-96aa-4f22-a45b-db8f0588e3c8)
![image](https://github.com/filamentphp/filament/assets/79960348/6996efbb-f209-499c-87f0-7c95b58b5355)

**Comment by sinnbeck** [comment]
So if you go to the tab and hit F5 it loads instantly?

**Reply by sinnbeck** [reply]
Be sure you have an index on that column it is filtering by. It is best to get the query, and do an explain to see how it is executed. And then add indexes based on that. You might need a compound index.

**Reply by sluxzinc92** [reply]
yes..  is there a way to make the tab a redirection link or put a url on it? 
![image](https://github.com/filamentphp/filament/assets/79960348/c882c3fb-9941-442e-9910-6e8ce064ed86)

**Reply by sinnbeck** [reply]
I dont think that is possible without a rewrite of the tabs code in filament as it does not allow you to pass the $tag type

Do you see the same slowdown in the official demo ?
https://demo.filamentphp.com/shop/orders

**Reply by sluxzinc92** [reply]
The official demo loads fast. I guess it's because it doesn't have much of conditional formatings. Mine has conditional formatings and customization like badges, color, to columns, tabs and action buttons..
